Resolve ESLint warnings in src/styles.js

### Overview
This file has ESLint errors or warnings:

```
src/styles.js
   9:40  warning  Missing trailing comma  comma-dangle
  12:44  warning  Missing trailing comma  comma-dangle
  13:8   warning  Missing trailing comma  comma-dangle
  19:41  warning  Missing trailing comma  comma-dangle
  22:44  warning  Missing trailing comma  comma-dangle
  23:8   warning  Missing trailing comma  comma-dangle
  29:44  warning  Missing trailing comma  comma-dangle
  30:8   warning  Missing trailing comma  comma-dangle
  36:44  warning  Missing trailing comma  comma-dangle
  37:8   warning  Missing trailing comma  comma-dangle
  40:58  warning  Missing trailing comma  comma-dangle
  45:44  warning  Missing trailing comma  comma-dangle
  46:8   warning  Missing trailing comma  comma-dangle
  51:44  warning  Missing trailing comma  comma-dangle
  52:8   warning  Missing trailing comma  comma-dangle
  53:6   warning  Missing trailing comma  comma-dangle
  54:4   warning  Missing trailing comma  comma-dangle
```

### Action Items
Resolve the errors or warnings. If there is a good reason to keep any piece of offending code, disable ESLint as shown below.

### Resources/Instructions
To see all ESLint errors and warnings, run this from the command line: `npm run lint`
Or for only one file, run this: `npx eslint path/to/file.js`

To close this issue by disabling errors or warnings when there is a reason that they cannot be resolved, use comments. 
To disable a rule for the whole file, put this at the top of the file:
`/* eslint-disable rule-name */`
To disable a rule for one line, put this right before the line:
`// eslint-disable-next-line rule-name`
Or put this at the end of the line:
`// eslint-disable-line rule-name`
Additional commenting options can be found in the [ESLint docs](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments).

### Dependency
This issue is one of many new issues this morning intended to replace the multi-file issue #169 with one issue per file.

--- a/src/styles.js
+++ b/src/styles.js
@@ -6,52 +6,52 @@ const useStyles = makeStyles(theme => ({
       backgroundColor: theme.palette.background.dark,
       padding: theme.spacing(2),
       '& h6': {
-        color: theme.palette.text.light
+        color: theme.palette.text.light,
       },
       '& .MuiTypography-body1': {
-        color: theme.palette.text.secondary
-      }
+        color: theme.palette.text.secondary,
+      },
     },
     '.card241': {
       backgroundColor: theme.palette.background.primary,
       padding: theme.spacing(2),
       '& h6': {
-        color: theme.palette.text.bright
+        color: theme.palette.text.bright,
       },
       '& .MuiTypography-body1': {
-        color: theme.palette.text.secondary
-      }
+        color: theme.palette.text.secondary,
+      },
     },
     '.card242': {
       backgroundColor: theme.palette.background.light,
       padding: theme.spacing(2),
       '& h6': {
-        color: theme.palette.secondary.dark
-      }
+        color: theme.palette.secondary.dark,
+      },
     },
     '.card244': {
       backgroundColor: theme.palette.background.secondary,
       padding: theme.spacing(2),
       '& h6': {
-        color: theme.palette.secondary.dark
-      }
+        color: theme.palette.secondary.dark,
+      },
     },
     '.frame223': {
-      backgroundColor: theme.palette.background.secondary
+      backgroundColor: theme.palette.background.secondary,
     },
     '.frame222': {
       backgroundColor: theme.palette.background.dark,
       '& p': {
-        color: theme.palette.text.secondary
-      }
+        color: theme.palette.text.secondary,
+      },
     },
     '.frame226': {
       backgroundColor: theme.palette.background.primary,
       '& p': {
-        color: theme.palette.text.secondary
-      }
-    }
-  }
+        color: theme.palette.text.secondary,
+      },
+    },
+  },
 }))
 
 export default useStyles

